### DISCUSSION TITLE
Error message when saving my own jupyter notebook

### DISCUSSION THREAD

**Discussion by GregCarr658** [cooperrc/computational-mechanics#191] [Q&A]
Whenever I go to save the notebook to my work folder so I can access later, I get this error message. I hit save as delete the old prompt and enter this as my file name. I can save it as a PDF to submit in google classroom but can't save my own version of the notebook.
![image](https://github.com/cooperrc/computational-mechanics/assets/135034891/3b67d1d5-1307-415c-a54d-392bc7396096)

**Comment by ewk19001** [comment]
I get the same error

**Comment by cooperrc** [comment]
try adding `.md` instead of letting the system default to `.ipynb`

**Reply by GregCarr658** [reply]
that solved the issue Thank you.
